src/Application/Configuration.php

```diff
@@ -26,12 +26,18 @@ class Configuration
      */
     private $version;
 
-    public function __construct(string $key, string $username, string $endpoint, string $version = '1.0')
+    /**
+     * @var string|null
+     */
+    private $source;
+
+    public function __construct(string $key, string $username, string $endpoint, string $version = '1.0', ?string $source = 'SDK')
     {
         $this->key = $key;
         $this->username = $username;
         $this->endpoint = $endpoint;
         $this->version = $version ?? '1.0';
+        $this->source = $source ?? 'SDK';
```

Comment:
Isn't it redundant if you are already initializing the source in the method parameters?

Reply:
Fixed. Thanks!